adagrams/game.py

```diff
@@ -1,11 +1,95 @@
+# Wave 1 
+import random
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    letters = []
+    all_letter_pool = []
+    for letter in LETTER_POOL:
+        letter_freq = LETTER_POOL[letter]
+        all_letter_pool += letter * letter_freq
+
+    for i in range(10):
+        letter_choice = random.choice(all_letter_pool)
+        letters.append(letter_choice)
+        all_letter_pool.remove(letter_choice)
+    return letters
 
+#wave 2
 def uses_available_letters(word, letter_bank):
-    pass
+    letter_bank_copy = letter_bank[:]
+    for letter in word.upper():
+        if letter in letter_bank_copy:
+            letter_bank_copy.remove(letter)
+    if len(letter_bank) - len(word) == len(letter_bank_copy):
+        return True
+    return False
```

Comment:
You could simplofy this conditonal by checking adding an else statement instead. Something like this:
```
for letter in word.upper():
        if letter in letter_bank_copy:
            letter_bank_copy.remove(letter)
        else:
            return False
```